### PR TITLE
WX - Issue "Functionality" #7: Hide Items After Splitting Bills

### PR DESCRIPTION
**Pull request by InariHimeko**
Now after splitting bills, the items are hidden rather than deleted. A new section at the bottom of the home screen includes all the hidden items.

### CODE REVIEW

**Comment by lawrencewang30** [comment]
The dropdown menu shows all the split items. LGTM after fixing the merge conflicts

**Review by kenisha-v — APPROVED** [review]
LGTM

**Pull request merged**
